### PR TITLE
chore: dismiss all before returning home

### PR DESCRIPTION
**Pull request by im-adithya**
This removes the back button in the header, if you cancel/finish setup

Note: for some reason, this cannot be observed in Expo Go but the production build also has this issue.

## Screenshots

<table><tr><td>

</td><td>

</td></tr></table>

### CODE REVIEW

**Review by rolznz — APPROVED** [review]
utACK

File: pages/settings/wallets/SetupWallet.tsx [hunk 1]
```diff
@@ -119,7 +119,10 @@ export function SetupWallet() {
       position: "top",
     });
 
-    router.replace("/");
+    if (router.canDismiss()) {
+      router.dismissAll();
+    }
+    router.navigate("/");
```
Comment (rolznz) [inline pages/settings/wallets/SetupWallet.tsx:125]: is this navigation even needed if we dismiss all the routes? shouldn't this be the default?

Comment (rolznz) [reply pages/settings/wallets/SetupWallet.tsx:125]: Won't this also push a route onto the stack, leaving a back button?

**Review by rolznz — CHANGES_REQUESTED** [review]
EDIT: please check comments

**Comment by im-adithya** [comment]
Addressed, should be good now!

**Pull request merged**